airbyte-db/db-lib/src/test/java/io/airbyte/db/util/SSLCertificateUtilsTest.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.nio.file.FileSystem;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+import java.security.spec.InvalidKeySpecException;
+import org.junit.jupiter.api.Test;
+
+class SSLCertificateUtilsTest {
+
+  private static final String SLASH_TMP = "/tmp";
+  private static final String KEY_STORE_PASSWORD = "123456";
+  private static final String KEY_STORE_PASSWORD2 = "78910";
+  static final String caPem = "-----BEGIN CERTIFICATE-----\n"
```

Comment:
please just make sure these aren't any of your or company's real private keys :)

Reply:
It's all play keys and certs I created.
My real passwords are way harder to guess (2468…)